packages/api/routes/index.js

```diff
@@ -10,4 +11,7 @@ router.get('/nomination/:id', nominationController.getNominationById);
 // user endpoints
 router.get('/user/:id', userController.getUserById);
 
+// grant cycle endpoints
+router.post('/grantcycle/new', grantCycleController.createGrantCycle);
```

Comment:
Generally would just make this `/grantcycle` for a restful API. The `POST` implies creation; you don't need the `new` bit.